docs/flatcar-linux/bare-metal.md

```diff
@@ -142,7 +142,7 @@ Configure the Matchbox provider to use your Matchbox API endpoint and client cer
 
 ```tf
 provider "matchbox" {
-  version     = "0.2.3"
+  version     = "0.3.0"
```

Comment:
Does this version work with terraform 0.11? If compatibility is not kept, maybe is better to do this atomically in the commit that upgrades to terraform 0.12. If not, the tree may be completely broken in this commit

Reply:
Still works with Terrraform 0.11: https://github.com/poseidon/terraform-provider-matchbox/blob/master/CHANGES.md#v030